core/templates/dev/head/domain/collection/GuestCollectionProgressService.js

```diff
@@ -62,33 +62,16 @@ oppia.factory('GuestCollectionProgressService', [
       });
     };
 
-    // This method corresponds to collection_domain.get_next_exploration_ids.
+    // This method corresponds to collection_domain.get_next_exploration_id.
     var _getNextExplorationIds = function(collection, completedIds) {
```

Comment:
Decide whether plural or singular (doesn't match comment above).

Reply:
Done, changed all to singular.